pandas/core/dtypes/common.py

```diff
@@ -107,11 +109,16 @@ def ensure_int64_or_float64(arr, copy=False):
     out_arr : The input array cast as int64 if
```

Comment:
make this simpler; these statements are for the Notes section

Reply:
done (moved what I added to the notes section)